### ISSUE TITLE
Automatic pasting into another voice

### ISSUE THREAD

**Issue by julenComposer** [musescore/MuseScore#20518]
### Your idea

Dear all, I propose you to consider a new feature for pasting a certain passage into a different voice, without using voice-exchange + selection filter workarounds.

### Problem to be solved

Currently you can use the voice swap-tool to re-assign the voice playing your materials. However, when I have a new idea that requires me to copy-paste some previous elements into a different voice, currently the only option seems is to use a combination of voice-swap function and selection filter.
This might add some firction to the workflow, and also I have found that with less performative machines (or even when working on large files) this can temporarily freeze the software, which is always a bit unsettling.

**Comment by wizofaus** [comment]
Pasting a selection from one voice into another voice is something I used to do all the time with Encore when I used that, and definitely a feature I've missed in MuseScore, particularly because of some very curious "design limitations" with the exchange voices and "use voice x" commands (particular wrt partial measures and/or tuplets).

**Comment by Reedbeta** [comment]
This would save me a lot of time when editing orchestral scores that have similar material between different voices of different staves.

**Comment by JJN76** [comment]
Yes, a thousand times. This is one of those things that even though there is a technically possible way to do it (exchange voices), it requires a change of mental gears from composition, so breaks flow. What would help with scoping development on this?

**Comment by cbjeukendrup** [comment]
I certainly think this will be done, but it is one of the _many_ open issues and requests, so it can't be prioritised immediately. 

A question that immediately comes to my mind is what should happen when the copied source material contains multiple voices.

**Comment by JJN76** [comment]
The multi-voice paste question is important to think about -- my first take is that if this overwrites all subsequent voices (e.g. Copy voice 1+2 part A, paste into voice 2 part B --> overwrite voice 2+3 part B, etc.) it is at least intuitive. Effectively you can take a string of music from any voice, paste into any other voice and it will just cascade from there. 

If my use case is typical, though, that problem would be atypical. I can't think of a time I wanted this for copying and pasting multiple voice groupings--I think it's always one to one. For example, if I decide to double the first and second violins with the clarinets, but have the clarinets written as two voices on a single staff, I'd like to be able to copy violin I to clarinet I (voice 1) and violin II to clarinet II (voice 2) directly.
